docs/METHODS_RPC.md

```diff
@@ -0,0 +1,107 @@
+## RPC
+
+_The following RPC methods are not part of the default Substrate runtime. They are hard-coded in Substrate._
```

Comment:
Ok, this is a bit wrong. The RPCs are never exposed by the runtime, rather they are provided by substrate itself. So these are the Remote Calls that are available by default. (It allows you to interact with the actual node, query and submit)